### DISCUSSION TITLE
Edge error

### DISCUSSION THREAD

**Discussion by drdedou** [seleniumbase/SeleniumBase#2590] [Q&A] [answered]
when I start Edge browser with privilege Administration I get this error 
![Screenshot 2024-03-10 145205](https://github.com/seleniumbase/SeleniumBase/assets/47856801/1d9ed93c-c6d2-4a6c-97c7-4c8d1e2c7360)

But it works perfectly when run in default user

**Comment by mdmintz** [comment, marked as answer]
1. You already asked the same question a few hours ago. I already told you that UC Mode doesn't support Edge.

<img width="540" alt="Screenshot 2024-03-10 at 7 42 43 PM" src="https://github.com/seleniumbase/SeleniumBase/assets/6788579/d28d6152-9f52-4f53-a476-85ad54a588a3">

2. If you have permissions issues when using `PyInstaller` to package your script as an executable, then that is an issue with PyInstaller: https://github.com/pyinstaller/pyinstaller/issues (not SeleniumBase).

3. And if you open tickets, you need to provide full steps to reproduce the issue.
